docs/articles/documentation/using-testcafe/common-concepts/screenshots-and-videos.md

```diff
@@ -0,0 +1,297 @@
+---
+layout: docs
+title: Screenshots and Videos
+permalink: /documentation/using-testcafe/common-concepts/screenshots-and-videos.html
+---
+# Screenshots and Videos
+
+TestCafe allows you to take screenshots of the webpage under test and record videos of test runs.
```

Comment:
IFAIK we have a common limitation about making screenshots and videos on remote browsers (see [here](https://devexpress.github.io/testcafe/documentation/test-api/actions/take-screenshot.html)). I think we can place it in this section and add a cross-link to the `Take Screenshot` action` section.

Reply:
we can also mention about limitation for video recording: we can do it only for Chrome and Firefox

Reply:
> we can do it only for Chrome and Firefox

wow, how come I didn't know it till now?

Reply:
idk)